### PR TITLE
Changed permissions of steering role to be PrivLevelSteering

### LINKED ISSUE #3506: Role changed from steering to operations for the endpoints used to manage steering targets
https://github.comcast.com/cdn/trafficcontrol/blob/deploy-20190102/traffic_ops/traffic_ops_golang/routes.go#L406-L408

```
{1.1, http.MethodPost, `steering/{deliveryservice}/targets/?(\.json)?$`, api.CreateHandler(&steeringtargets.TOSteeringTargetV11{}), auth.PrivLevelOperations, Authenticated, nil},
{1.1, http.MethodPut, `steering/{deliveryservice}/targets/{target}/?(\.json)?$`, api.UpdateHandler(&steeringtargets.TOSteeringTargetV11{}), auth.PrivLevelOperations, Authenticated, nil},
{1.1, http.MethodDelete, `steering/{deliveryservice}/targets/{target}/?(\.json)?$`, api.DeleteHandler(&steeringtargets.TOSteeringTargetV11{}), auth.PrivLevelOperations, Authenticated, nil},
```

^^ those routes in perl required a privlevel of 15 (portal role) not 20 (for the operations role).

suggestion: change the role to steering since steering is also 15 and is a bit more clear.

### PR DESCRIPTION
**Pull request by moltzaum**
## What does this PR (Pull Request) do?

- [x] This PR fixes #3506

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?
Log in as a steering role. Make sure you can access the steering endpoints.

## If this is a bug fix, what versions of Traffic Ops are affected?
master

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

**Comment by mitchell852** [comment]
ideally, these tests: https://github.com/apache/trafficcontrol/blob/master/traffic_ops/testing/api/v14/steeringtargets_test.go

would run using a "steering" user to ensure that users w/ the steering role can do these actions. But I'm not sure the testing framework is setup to use different users for different tests currently.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3547/
Test PASSed.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3558/
Test PASSed.

File: traffic_ops/testing/api/v14/withobjs.go [hunk 1]
```diff
@@ -85,9 +85,9 @@ var withFuncs = map[TCObj]TCObjFuncs{
 	Servers:                        {CreateTestServers, DeleteTestServers},
 	Statuses:                       {CreateTestStatuses, DeleteTestStatuses},
 	StaticDNSEntries:               {CreateTestStaticDNSEntries, DeleteTestStaticDNSEntries},
-	SteeringTargets:                {CreateTestSteeringTargets, DeleteTestSteeringTargets},
+	SteeringTargets:                {SetupSteeringTargets, DeleteTestSteeringTargets},
 	Tenants:                        {CreateTestTenants, DeleteTestTenants},
 	Types:                          {CreateTestTypes, DeleteTestTypes},
 	Users:                          {CreateTestUsers, ForceDeleteTestUsers},
-	UsersDeliveryServices:          {CreateTestUsersDeliveryServices, DeleteTestUsersDeliveryServices},
```
Comment (moltzaum) [inline traffic_ops/testing/api/v14/withobjs.go]: I'm not sure why, but gofmt re-indented this.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3559/
Test PASSed.

**Pull request merged**